tests/execute/restraint/report_result/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "output=\$(mktemp)" 0 "Create output file"
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "set -o pipefail"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Verify mocked Restraint rstrnt-report-result file generated correctly and read by tmt.'
+        rlRun "tmt run -vvvddd --remove | tee $output"
+	rlAssertGrep "Rstrnt-report-result output file detected." $output
```

Comment:
Please, do not mix tabs with spaces, it causes wrong indent of the code.

Reply:
Coming from a Python background, I never use the tab key always the space bar. Vim must have automatically added a tab character when I added a newline. I've corrected that in my most recent commit.